node/resolvers/oms/index.ts

```diff
@@ -6,5 +7,12 @@ export const queries = {
     return isUserLoggedIn(ctx) ? oms.userLastOrder() : null
   },
 
-  order: (_: any, {id}: {id: string}, {clients: {oms}}: Context) => oms.order(id)
+  order: (_: any, { id }: { id: string }, { clients: { oms } }: Context) => oms.order(id)
+}
+
+export const fieldResolvers = {
+  OrderItemPaymentConnectorResponse: {
+    meoWalletReference: propOr(null, 'mb.reference'),
```

Comment:
And here change it to:

```ts
  additionalData: ({ tid, returnCode, message, ...rest }) => rest
```

Reply:
I needed to apply default values to object destructuring to skip unknown fields.